topic-operator/src/test/java/io/strimzi/operator/topic/v2/TopicControllerIT.java

```diff
@@ -1904,4 +1955,27 @@ public void shouldTerminateIfQueueFull(
         operator = null;
         maybeStartOperator(topicOperatorConfig("ns", kafkaCluster));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void shouldSkipReconciliationsWhenKafkaTopicIsPaused(
```

Comment:
We should add tests that deleted paused CRs do get reconciled and one covering CRs created in the paused state.

Reply:
Sure.